site/gatsby-site/content/about/index.es.mdx

```diff
@@ -41,9 +41,60 @@ La base de datos de incidentes es administrada de manera participativa por perso
+* **[Kit Harris](https://www.linkedin.com/in/kitharris/):** Kit dirige investigaciones de subvenciones e investiga campos prometedores en Longview Philanthropy. También escribe sobre las recomendaciones e investigaciones de Longview para filántropos clientes de Longview. Antes de centrarse en el trabajo filantrópico de alto impacto, Kit trabajó como comerciante de derivados de crédito con J.P. Morgan. Durante ese tiempo, donó la mayoría de sus ingresos a organizaciones benéficas de alto impacto. Kit tiene un título de primera clase en matemáticas de la Universidad de Oxford.
+**Contribuciones:** Kit sirve como observador de la junta, brinda asesoramiento estratégico y es el punto de contacto de RAIC en Longview.
+
+
+# Colaboradores
+
+**Contribuidores de código abierto:** personas que han contribuido con más de una solicitud de incorporación de cambios, gráficos, copias del sitio o informes de errores a la base de datos de incidentes de AI.
+
+* [César Varela](https://github.com/cesarvarela)
+* [Alex Musca](https://github.com/alexmcode)
+* [Chloe Kam](http://kamchy.com) (Logotipos AIID)
+* [JT McHorse](https://github.com/jt-mchorse)
+* Seth Rey
+
+**Asociación en miembros del personal de AI:**
+[Jingying Yang](https://www.linkedin.com/in/jingyingyang/) y [Dr. Christine Custis](https://www.su.edu/symposium/business-symposium-speakers/christine-custis/) contribuyó significativamente a las primeras etapas del AIID.
+
+**Editores de incidentes:** personas que resuelven los envíos de incidentes a la base de datos.
+
+* [Sean McGregor](https://seanbmcgregor.com/)
+
+**Editores de taxonomías:** Organizaciones o personas que han contribuido [taxonomías](https://incidentdatabase.ai/taxonomies) a la base de datos.
+
+* [Centro de Seguridad y Tecnología Emergente](https://cset.georgetown.edu/) (CSET)
+
+**Contribuyentes de incidentes:** personas que han contribuido con una gran cantidad de incidentes a la base de datos.
+
+* [Kate Perkins (Intel)](/apps/discover?display=detalles&lang=en&page=1&submitters=Kate%20Perkins)
+* [Roman Lutz (Instituto Max Planck para Sistemas Inteligentes, anteriormente Microsoft)](/apps/discover?display=details&lang=en&page=1&submitters=Roman%20Lutz)
+* [Patrick Hall (Burt and Hall LLP)](/apps/discover?display=details&lang=en&page=1&submitters=Patrick%20Hall%20%28BNH.ai%29)
+* [Catherine Olsson (Google)](/apps/discover?display=detalles&lang=en&page=1&submitters=Catherine%20Olsson)
+* [Roman Yampolskiy (Universidad de Louisville)](/apps/discover?display=detalles&lang=en&page=1&submitters=Roman%20Yampolskiy)
+* Sam Yoon (como contratista de PAI, luego con Deloitte Consulting, luego con la Escuela de Gobierno Kennedy)
+
+Las siguientes personas han recopilado una gran cantidad de incidentes que están pendientes de ingesta.
+
+* Zachary Arnold, Helen Toner, Ingrid Dickinson, Thomas Giallella y Nicolina Demakos (Centro de Seguridad y Tecnología Emergente, Georgetown)
```

Comment:
Change `Centro de Seguridad y Tecnología Emergente` to `Center for Security and Emerging Technology`